frontend/public/components/overview/project-overview.tsx

```diff
@@ -188,11 +176,14 @@ const Alerts: React.SFC<AlertsProps> = ({item}) => {
     return null;
   }
 
-  const { error, warning, info } = _.groupBy(alerts, 'severity');
+  const { error, warning, info, buildNew, buildPending, buildRunning, buildFailed, buildError } = _.groupBy(alerts, 'severity');
```

Comment:
Instead of prefixing the severity with `build` I think it should be a separate object with just the build alerts. Or it could be another `type` property on the alert object